woocommerce/woocommerce-ios
# Add screenshot dark mode

Adds Dark Mode screenshots!

**To Test:**
1. Check out this PR
2. Run `bundle exec fastlane screenshots`
3. Note that there are now dark + light versions of each screenshot!


Update release notes:
